prowler/lib/cli/parser.py

```diff
@@ -343,14 +350,25 @@ def __init_aws_parser__(self):
             default=None,
             help="Path for allowlist yaml file. See example prowler/config/allowlist.yaml for reference and format. It also accepts AWS DynamoDB Table or Lambda ARNs or S3 URIs, see more in https://docs.prowler.cloud/en/latest/tutorials/allowlist/",
         )
-        # Allowlist
-        audit_tags_subparser = aws_parser.add_argument_group("Tags-based Scan")
-        audit_tags_subparser.add_argument(
+        # Based Scans
+        aws_based_scans_subparser = aws_parser.add_argument_group("AWS Based Scans")
+        aws_based_scans_parser = (
+            aws_based_scans_subparser.add_mutually_exclusive_group()
+        )
+        aws_based_scans_parser.add_argument(
             "-t",
             "--scan-tags",
             nargs="+",
             default=None,
-            help="Scan only resources with specific tags (Key=Value), e.g., Environment=dev Project=prowler",
+            help="Scan only resources with specific AWS Tags (Key=Value), e.g., Environment=dev Project=prowler",
+        )
+        aws_based_scans_parser.add_argument(
+            "-r",
+            "--resource-arns",
+            nargs="+",
+            type=arn_type,
```

Comment:
What is this?

Reply:
It is a custom type that calls the `arn_type` function to validate each input ARN.

Reply:
awesone, even with a list?

Reply:
yeah 👍🏼